src/Sylius/Bundle/CoreBundle/CatalogPromotion/Command/RemoveInactiveCatalogPromotion.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\CatalogPromotion\Command;
+
+final class RemoveInactiveCatalogPromotion
```

Comment:
I don't have strong opinion but `RemoveCatalogPromotion` should be enough, information about `inactive` is an implementation detail, and IMO can be ommit.

Reply:
For now, I have left `RemoveInactiveCatalogPromotion`. According to https://github.com/Sylius/Sylius/pull/14270/files#r970658689
if we introduce the aforementioned criteria checker, I'm in favor of changing it  🚀